src/Apps/Sell/Routes/TitleRoute.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react"
+import * as Yup from "yup"
+import { Input, Text } from "@artsy/palette"
+import { TitleRoute_submission$key } from "__generated__/TitleRoute_submission.graphql"
+import { graphql, useFragment } from "react-relay"
+import { Formik, useFormikContext } from "formik"
+import { DevDebug } from "Apps/Sell/Components/DevDebug"
+import { BottomFormNavigation } from "Apps/Sell/Components/BottomFormNavigation"
+import { useArtworkFormContext } from "Apps/Sell/ArtworkFormContext"
+
+const FRAGMENT = graphql`
+  fragment TitleRoute_submission on ConsignmentSubmission {
+    title
+  }
+`
+
+const Schema = Yup.object().shape({
+  title: Yup.string().required().trim(),
+})
+
+interface FormValues {
+  title: string
+}
+
+const InnerForm: React.FC = props => {
+  const { values, handleChange } = useFormikContext<FormValues>()
+
+  return (
+    <>
+      <Input
+        onChange={handleChange}
+        required
+        name="title"
+        title="Title"
+        defaultValue={values.title} />
+    </>
+  )
+}
+
+interface TitleRouteProps {
+  submission: TitleRoute_submission$key
+}
+
+export const TitleRoute: React.FC<TitleRouteProps> = props => {
```

Comment:
One specific bit of feedback I'd like in this PR is on the structure/separation of these steps. I think it's valuable that each step be isolated/independent from others and each of the steps in this PR:
- Query for data scoped to the step
- Validate against form inputs scoped to the step
- Mutate data scoped to the step

This will make it easier to introduce new steps in the flow and eventually group steps into different phases, exposed depending on the state of the submission. For example we request and collect information in a Tier 1 group right away and plan to only request and collect information in a Tier 2 group of information gathering after the submission has been approved.

Reply:
Yep — we do a lot of these stepped wizard things. You want completely isolated steps wherein they have no knowledge of how they might be arranged. Then a parent context that controls the arrangement, provides next/prev functions, and accumulates any state.

When they are linear that arrangement is just expressed as an array or a numeric index. When they might branch based on conditions we use the workflow engine. [Example](https://github.com/artsy/force/blob/main/src/Components/Onboarding/config.ts).

Reply:
We might have some conditional needs for steps in the future so might reach for the workflow engine later.